circuits/circom/batchUpdateStateTree.circom

```diff
@@ -54,7 +54,7 @@ template BatchUpdateStateTree(
         // if msg_tree_batch_end_index < message_indices[i], use msg_tree_batch_end_index
         // if msg_tree_batch_end_index >= message_indices[i], use message_indices[i]
 
-        message_indices[i] = msg_tree_batch_start_index + i;
+        message_indices[i] <== msg_tree_batch_start_index + i;
```

Comment:
Hummm, the reason for this diff is zkutils complaining this
```
ERROR at /Users/liangcc/projects/EAPL/semaphore_projects/maci/circuits/circom/batchUpdateStateTree.circom:57,8-57,59   Cannot assign to a signal with `=` use <-- or <== ops
```

Reply:
Could it be a different version of `circom`?

```bash
~/maci/circuits$ rm build/batchUstCircuit.json && ./scripts/buildBatchUpdateStateTreeSnark.sh
Compiling circom/prod/batchUpdateStateTreeVerifier.circom...
Generated build/batchUstCircuit.json
Params file exists. Skipping setup.
Generating Solidity verifier...
Copying BatchUpdateStateTreeVerifier.sol to contracts/sol.
~/maci/circuits$ npx circom --version
0.0.34
```

Reply:
Thanks, let me check

Reply:
humm, I'm using the same version too.
```
$  npx circom --version
0.0.34
```